virttest/virsh.py

```diff
@@ -3329,6 +3329,33 @@ def snapshot_list(name, options=None, **dargs):
     return ret
 
 
+def snapshot_status(name, options=None, **dargs):
+    """
+    Get list of snapshots status of domain.
+
+    :param name: name of domain
+    :param options: options of snapshot_list
+    :param dargs: standardized virsh function API keywords
+    :return: dict of snapshot names and snapshot status
+    """
+    # CmdResult is handled here, force ignore_status
+    dargs["ignore_status"] = True
+    status_dict = {}
+
+    cmd = "snapshot-list %s" % name
+    if options is not None:
+        cmd += " %s" % options
+    sc_output = command(cmd, **dargs)
+    if sc_output.exit_status != 0:
+        raise process.CmdError(cmd, sc_output, "Failed to get list of snapshots")
```

Comment:
Can it be covered by setting 'dargs["ignore_status"] = False' ?

Reply:
Thanks yingshun ,You are right , updated